app/scripts/TiledPixiTrack.js

```diff
@@ -224,12 +224,6 @@ class TiledPixiTrack extends PixiTrack {
     });
   }
 
-  setError(error) {
-    this.errorTextText = error;
-    this.draw();
-    this.animate();
```

Comment:
Noticing that this `setError` impl that we removed has the side effect of `draw()` + `animate()` while the new base impl just calls `PixiTrack.drawError()`. Are these equivalent?

Reply:
Well the previous draw used to just call `drawError` so we're basically cutting out the middleman.

Reply:
The lack of the animate call is slightly more concerning but I've been using this function a fair amount in resglass and haven't noticed any issues so I'm inclined to leave it out.